MatrixSDK/NotificationCenter/ServiceTerms/MXServiceTerms.m

```diff
@@ -18,6 +18,7 @@
 #import "MXServiceTermsRestClient.h"
 
 #import "MXRestClient.h"
+#import "MXIdentityServerRestClient.h"
```

Comment:
useless import, no?

Reply:
No, it's needed for `kMXIdentityAPIPrefixPathV2` constant.